addons/godot-next/global/editor_tools.gd

```diff
@@ -3,6 +3,8 @@
 # brief_description: A utility for any features useful in the context of the Editor
 # attributions:
 # - 'is_in_edited_scene' sourced from @Zylann
+# - 'generate_group' sourced from @xDGameStudios
```

Comment:
I actually need to move all of these attributions to the ATTRIBUTIONS.md file. Either you can move them yourself, or just ignore it and I can do it after merging this PR.

Reply:
Removed my name from the `editor_tools.gd`. Didn't touch the `ATTRIBUTIONS.md` file, will leave that to you ;)